src/Annotator.js

```diff
@@ -275,15 +284,24 @@ class Annotator extends EventEmitter {
             </div>`.trim();
 
         this.container.appendChild(this.mobileDialogEl);
+    }
 
-        const pointController = this.modeControllers[TYPES.point];
-        if (pointController) {
-            pointController.setupSharedDialog(this.container, {
-                isMobile: this.isMobile,
-                hasTouch: this.hasTouch,
-                localized: this.localized
-            });
+    /**
+     * Hides and resets the shared mobile dialog.
+     *
+     * @return {void}
+     */
+    resetMobileDialog() {
+        if (!this.mobileDialogEl || this.mobileDialogEl.classList.contains(CLASS_HIDDEN)) {
+            return;
         }
+
+        // Reset the mobile dialog
+        util.hideElement(this.mobileDialogEl);
+        this.mobileDialogEl.innerHTML = `
```

Comment:
What're team thoughts on `innerHTML`? I'd love us to move away from it, due to the "performance implications", and instead have us create/modify existing nodes. My preference is to create a documentFragment, and insert that into the parent element (in this case `this.mobileDialogEl)

@jeremypress @pramodsum @tonyjin @DanDeMicco

Reply:
Agreed. I moved a lot of the annotations codebase over to this a while back too and might have missed this one. I'll make a separate PR later to clear up any other instances :)

Reply:
+1 for document fragment, especially for when you're manipulating something something multiple times to generate a final DOM structure.